Makefile

```diff
@@ -11,7 +11,7 @@ DOCKER_SBOM_PLUGIN_VERSION=0.6.1
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= $(shell git describe --tags --dirty --broken | cut -c 2-)
+VERSION ?= $(shell git describe --always --tags --dirty --broken | cut -c 2-)
```

Comment:
what does `--always` change in terms of VERSION generation?

Reply:
This makes sure that `git` always returns exit code 0. Without it, if will fail with "Nothing to describe" if there are no tags